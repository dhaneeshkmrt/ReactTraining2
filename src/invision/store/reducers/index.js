import { combineReducers } from "redux";

import { movieReducer } from "./movie.reducer";
import { popupReducer } from "./popup.reducer";

export const reducers = combineReducers({
  movies: movieReducer,
  popups: popupReducer
})

