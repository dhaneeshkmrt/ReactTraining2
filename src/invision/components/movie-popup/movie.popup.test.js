import { store } from "../../store";
import React from "react";
import { Provider } from 'react-redux';

import MoviePopup from "./movie.popup";
import TestRenderer from 'react-test-renderer';


describe('Movie Popup: ', () => {
  
  it('should render', ()=>{
    const searchBar = TestRenderer.create(<Provider store={store}><MoviePopup /></Provider >);
    const json = searchBar.toJSON();
    expect(json).toMatchSnapshot();
  })
});