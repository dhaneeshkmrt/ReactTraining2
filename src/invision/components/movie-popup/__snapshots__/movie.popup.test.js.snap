// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Popup:  should render 1`] = `
<div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "height": "100%",
        "justifyContent": "center",
        "left": 0,
        "position": "fixed",
        "top": 0,
        "width": "100%",
        "zIndex": 10000,
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "#fff",
          "borderRadius": "5px",
          "boxShadow": "0 2px 4px rgba(0, 0, 0, 0.3)",
          "boxSizing": "border-box",
          "transform": "translate3d(0, 0, 0)",
          "transition": "transform 500ms cubic-bezier(0, 0, 0.25, 1), opacity 500ms cubic-bezier(0, 0, 0.25, 1)",
          "width": "600px",
          "zIndex": 10002,
        }
      }
    >
      <div
        className="add-movie-ctnr"
      >
        <div
          className="header-close-btn"
        >
          <a
            onClick={[Function]}
          >
            Ã—
          </a>
        </div>
        <h1 />
        <form
          className="form-ctnr"
          id="dktest"
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <div
            className="form-field"
          >
            <label
              htmlFor="title"
            >
              Title
            </label>
            <br />
            <input
              name="title"
              onChange={[Function]}
              placeholder="Title"
              type="text"
              value=""
            />
            <div
              className="form-field"
            >
              <label
                htmlFor="title"
              >
                Release Date
              </label>
              <br />
              <input
                name="releasedDate"
                onChange={[Function]}
                placeholder="Release Date"
                type="date"
                value=""
              />
              <div
                className="error-message"
              />
            </div>
            <div
              className="form-field"
            >
              <label
                htmlFor="title"
              >
                Movie URL
              </label>
              <br />
              <input
                name="thumbnail"
                onChange={[Function]}
                placeholder="Movie URL"
                type="text"
                value=""
              />
              <div
                className="error-message"
              />
            </div>
            <div
              className="form-field"
            >
              <label
                htmlFor="title"
              >
                GENRE
              </label>
              <br />
              <input
                name="genre"
                onChange={[Function]}
                placeholder="GENRE"
                type="text"
                value=""
              />
              <div
                className="error-message"
              />
            </div>
            <div
              className="form-field"
            >
              <label
                htmlFor="title"
              >
                OVERVIEW
              </label>
              <br />
              <input
                name="overview"
                onChange={[Function]}
                placeholder="OVERVIEW"
                type="text"
                value=""
              />
              <div
                className="error-message"
              />
            </div>
            <div
              className="form-field"
            >
              <label
                htmlFor="title"
              >
                Rating
              </label>
              <br />
              <input
                name="rating"
                onChange={[Function]}
                placeholder="OVERVIEW"
                type="text"
                value=""
              />
              <div
                className="error-message"
              />
            </div>
            <div
              className="form-field"
            >
              <label
                htmlFor="title"
              >
                RUNTIME
              </label>
              <br />
              <input
                name="runTime"
                onChange={[Function]}
                placeholder="RUNTIME"
                type="text"
                value=""
              />
              <div
                className="error-message"
              />
            </div>
            <div
              className="form-footer"
            >
              <button
                className="primary-btn reset-button"
                type="reset"
              >
                Reset
              </button>
              <button
                className="primary-btn"
                onClick={[Function]}
                type="submit"
              >
                Submit
              </button>
            </div>
          </div>
        </form>
      </div>
    </div>
    <div
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "rgba(0, 0, 0, 0.7)",
          "height": "100%",
          "left": 0,
          "position": "fixed",
          "top": 0,
          "width": "100%",
          "zIndex": 10001,
        }
      }
    />
  </div>
</div>
`;
